Makefile

```diff
@@ -5,6 +5,14 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 0.9.0
 
+# Try to detect Docker or Podman
+CONTAINER_RUNTIME := $(shell command -v docker 2> /dev/null || command -v podman 2> /dev/null)
+
+# If neither Docker nor Podman is found, print an error message and exit
+ifeq ($(CONTAINER_RUNTIME),)
+$(error "No container runtime (Docker or Podman) found in PATH. Please install one of them.")
+endif
```

Comment:
Can we remove this check or just log the message and continue without exiting? My concern with this is that if I just want to run some non image build command like `make run` or `make install` or some unit test on system without docker/podman, it won't allow me to do so.

Reply:
changed from error to warning, so now it should only print a warning instead of exiting with an error